cmd/common-main.go

```diff
@@ -153,6 +153,19 @@ func handleCommonEnvVars() {
 		globalCacheExpiry = expiry
 	}
 
+	if intervalStr := os.Getenv("MINIO_USAGE_CHECK_INTERVAL"); intervalStr != "" {
+		interval, err := time.ParseDuration(intervalStr)
```

Comment:
we could move parsing of a string to `time.Duration` and validating it into a separate function.